.dotnet/src/Generated/Models/CreateTranscriptionRequest.Serialization.cs

```diff
@@ -176,5 +184,76 @@ internal static CreateTranscriptionRequest FromResponse(PipelineResponse respons
             using var document = JsonDocument.Parse(response.Content);
             return DeserializeCreateTranscriptionRequest(document.RootElement);
         }
+
+        private MultipartFormDataContent _multipartContent = default;
+        internal async Task<BinaryContent> ToBinaryContentAsync(CancellationToken cancellationToken = default)
```

Comment:
TODO: given current approach, this would return a tuple of BinaryContent and RequestOptions